### PR TITLE
Client 2630

### PR DESCRIPTION
**Pull request by alstojanovic**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Review by hero101 — CHANGES_REQUESTED** [review]
After a discussion, we are going to redo this in a different way.

File: src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx [hunk 1]
```diff
@@ -2,15 +2,14 @@ import React, { FC, useCallback, useState } from 'react';
...
-import CalloutSummaryStep from './steps/CalloutSummaryStep/CalloutSummaryStep';
 import { CalloutCreationType } from './useCalloutCreation/useCalloutCreation';
+import { Box, Button } from '@mui/material';
+import { DialogActions, DialogContent, DialogTitle } from '../../../../common/components/core/dialog';
+import { LoadingButton } from '@mui/lab';
+import CampaignOutlinedIcon from '@mui/icons-material/CampaignOutlined';
+import CalloutForm from '../CalloutForm';
 
-export type CalloutDialogCreationType = {
+export type Callout = {
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx:12]: What was wrong with the type's name? It's not a Callout either.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx [hunk 2]
```diff
@@ -47,4 +47,4 @@ const CalloutAspectTemplateStep: FC<CalloutAspectTemplateStepProps> = ({ callout
     />
   );
 };
-export default CalloutAspectTemplateStep;
+export default CalloutAspectTemplate;
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx:50]: Why was this renamed?

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx [hunk 3]
```diff
@@ -2,16 +2,16 @@ import React, { FC } from 'react';
...
+} from '../../../../../hooks/generated/graphql';
+import { CalloutType } from '../../../../../models/graphql-schema';
+import { useHub } from '../../../../challenge/hub/HubContext/useHub';
+import AspectTemplatePreview from '../../../aspect/AspectTemplatePreview/AspectTemplatePreview';
 import { TemplateListWithPreview } from './TemplateListWithPreview';
-import { CalloutTemplateStepProps } from './CalloutTemplateStepProps';
+import { CalloutTemplateProps } from './CalloutTemplateProps';
 
-export interface CalloutAspectTemplateStepProps extends CalloutTemplateStepProps {}
+export interface CalloutAspectTemplateStepProps extends CalloutTemplateProps {}
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx:12]: You have not renamed the props.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx (hunk 2 above)
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx:50]: This component is not used anymore.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx [hunk 4]
```diff
@@ -37,4 +37,4 @@ const CalloutCanvasTemplateStep: FC<CalloutCanvasTemplateStepProps> = ({ callout
     />
   );
 };
-export default CalloutCanvasTemplateStep;
+export default CalloutCanvasTemplate;
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx:40]: Why is it renamed. It's not used anymore.

File: src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx [hunk 5]
```diff
@@ -60,44 +51,44 @@ const CalloutCreationDialog: FC<CalloutCreationDialogProps> = ({ open, onClose,
...
-            component={CalloutSummaryStep}
-            title={t('components.callout-creation.create-step.title')}
-            callout={callout}
-            onClose={handleClose}
-            onSaveAsDraft={handleSummarySaveAsDraft}
-            isCreating={isCreating}
-          />
-        </Steps>
-      </StepLayoutHolder>
+      <Box>
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx:62]: This Box is not required.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx [hunk 6]
```diff
@@ -2,16 +2,16 @@ import React, { FC } from 'react';
...
-import { CalloutTemplateStepProps } from './CalloutTemplateStepProps';
+} from '../../../../../hooks/generated/graphql';
+import { CalloutType } from '../../../../../models/graphql-schema';
+import { useHub } from '../../../../challenge/hub/HubContext/useHub';
+import CanvasPreview from '../../../canvas/CanvasPreview/CanvasPreview';
+import { CalloutTemplateProps } from './CalloutTemplateProps';
 import { TemplateListWithPreview } from './TemplateListWithPreview';
 
-export interface CalloutCanvasTemplateStepProps extends CalloutTemplateStepProps {}
+export interface CalloutCanvasTemplateStepProps extends CalloutTemplateProps {}
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx:12]: You have not renamed the props.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutTemplateProps.ts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import { CalloutSumaryProps } from '../../CalloutSummary';
```
Comment (hero101) [inline src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutTemplateProps.ts:1]: I believe this is not needed anymore.

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutTemplateProps.ts (hunk 7 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutTemplateProps.ts:1]: It's needed in the `CalloutCanvasTemplate` and `CalloutAspectTemplate`.

File: src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx (hunk 5 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutCreationDialog.tsx]: Removed.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx (hunk 6 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx]: Renamed.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx (hunk 4 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutCanvasTemplate.tsx:40]: It is going to be used when Callout templates are introduced. It will be used not as a step but as part of the Callout creation dialog. My logic was either delete it or prepare for future use.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx (hunk 2 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx:50]: It is going to be used when Callout templates are introduced. It will be used not as a step but as part of the Callout creation dialog. My logic was either delete it or prepare for future use.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx (hunk 3 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx]: Renamed.

File: src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx (hunk 2 above)
Comment (alstojanovic) [reply src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutAspectTemplate.tsx:50]: Callout templates preparation.

**Review by hero101 — APPROVED** [review]
